test/library/standard/GMP/powCheck.good

```diff
@@ -0,0 +1,8 @@
+2^-1 = 0
+2^-2 = 0
+4^-1 = 0
+4^-2 = 0
+1^-1 = 1
+1^-2 = 1
+-1^-1 = -1
+-1^-2 = -1
```

Comment:
Using `bc`, I get:

> -1^-2
> 1

Reply:
Oops, looks like I needed parenthesis.  I'll fix that

Reply:
Okay, fixed.  Have another look?